docs/reference/commandline/service_create.md

```diff
@@ -145,6 +145,32 @@ The following command creates a "global" service:
 $ docker service create --name redis_2 --mode global redis:3.0.6
 ```
 
+### Service constraint
+
+You can limit the set of nodes where a task can be scheduled by defining
+constraint expressions. Multiple constraints find nodes that satisfy every
+expression (AND match). Constraints can match node or engine attributes as
+follows:
+
+| node attribute | matches | example |
+|:------------- |:-------------| :---------------------------------------------|
+| node.id | node ID | `node.id == 2ivku8v2gvtg4`                               |
+| node.hostname | node hostname | `node.hostname != node-2`                    |
+| node.role | node role: manager | `node.role == manager`                      |
+| node.labels | user defined node labels | `node.labels.security == high`      |
+| engine.labels | Docker Engine's labels | `engine.labels.operatingsystem == ubuntu 14.04`|
+
+`engine.labels` apply to Docker Engine attributes like operating system,
```

Comment:
`Docker Engine attributes` -> `Docker Engine labels`. `labels` are widely used term in Docker daemon.